packages/block-library/src/table-of-contents/block.json

```
@@ -0,0 +1,17 @@
+{
+	"name": "core/table-of-contents",
+	"category": "common",
+	"attributes": {
+		"headings": {
+			"type": "array",
+			"source": "query",
+			"selector": ".blocks-table-of-contents-entry",
+			"default": [],
+			"query": {
+				"content": { "source": "text" },
+				"anchor": { "source": "attribute", "attribute": "href" },
+				"level": { "source": "attribute", "attribute": "data-level" }
+			}
+		}
+	}
+}
```

This should have a trailing newline, as per the [root `.editorconfig` file](https://github.com/WordPress/gutenberg/blob/master/.editorconfig).
```suggestion
}

```